Observer/Config/CronFrequency.php

```diff
@@ -29,7 +29,7 @@ class CronFrequency extends YotpoCoreCronFrequency
      * @var array <mixed>
      */
     protected $cronFrequency = [
-        'customers_sync_frequency' => ['config_path' => '','job_code' => 'yotpo_cron_smsbumb_customers_sync']
```

Comment:
@manaftryzens Is this a fix or just a typo (together with https://github.com/YotpoLtd/magento2-module-messaging/pull/72/files#diff-044dd0540d0b1a26871b5f18bfcbf3a1684560b35f94a0567918c7f97cb6eed1L24)

Reply:
Yes, It is a typo correction.